sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -693,4 +694,29 @@ case class HiveTableRelation(
   override def newInstance(): HiveTableRelation = copy(
     dataCols = dataCols.map(_.newInstance()),
     partitionCols = partitionCols.map(_.newInstance()))
+
+  override def simpleString(maxFields: Int): String = {
```

Comment:
The explain output of `HiveTableScanExec` does not have the same issue?

Reply:
> The explain output of `HiveTableScanExec` does not have the same issue?

No, only caused. by `HiveTableRelation`